.gitmodules

```diff
@@ -1,3 +0,0 @@
-[submodule "oni-lda-c"]
-	path = oni-lda-c
-	url = https://github.com/OpenNetworkInsight/oni-lda-c
```

Comment:
I'm not sure why to remove for the oni to spot change. I think this change corresponds to changing repos structure o renaming repos. This is eliminating the recursivity of ml and lda, right? 

My point is we never sync .gitmodules file. After repo restructure we can _git pull_ and everything should be in order.